CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+# Changelog
+All notable changes to the gem ibm-cloud-sdk-ruby will be documented here.
+
+## [0.1.3] - 2020-08-31
```

Comment:
Would prefer to have an "unreleased" section where we can append changes prior to a release then when a gem owner releases the gem and bumps the version they can update the changelog